### PR TITLE
Added ReloadOperatingSystem and unit testcase

### PR DESCRIPTION
**Pull request by mattcui**
After merge, please do Godeps update for the bosh-softlayer-cpi project, then I will send pull request to merge OS reload code in bosh-softlayer-cpi project. Thanks.

### CODE REVIEW

File: services/softlayer_virtual_guest.go [hunk 1]
```diff
@@ -68,6 +68,29 @@ func (slvgs *softLayer_Virtual_Guest_Service) CreateObject(template datatypes.So
 	return softLayer_Virtual_Guest, nil
 }
 
+func (slvgs *softLayer_Virtual_Guest_Service) ReloadOperatingSystem(instanceId int, template datatypes.Image_Template_Config) error {
+	parameter := [2]interface{}{"FORCE", template}
```
Comment (maximilien) [inline services/softlayer_virtual_guest.go]: Interesting so "FORCE" is part of the API... Did not know. Thanks

**Comment by maximilien** [comment]
Hi, @mattcui,

Looks good merging. Please submit the Godeps PR for CPI since I am traveling tomorrow and will only have an hour or so and will review but may not have time to update Godeps and run tests myself.

Thanks,

Max

**Comment by maximilien** [comment]
Also, think about how to add an integration test for this here 😊

File: services/softlayer_virtual_guest.go (hunk 1 above)
Comment (mattcui) [inline services/softlayer_virtual_guest.go]: There require to combine a request body like this:

{"parameters":["FORCE",{"imageTemplateId":"651103"}]}

FORCE means no need to manual confirmation

**Comment by mattcui** [comment]
I am writing unit testcase and integration testcase for OS reload feature in CPI project, which still needs some time, I may submit the pull request for CPI after you arrive in Beijing next Monday.

**Comment by maximilien** [comment]
Cool thanks @mattcui. Just arrive Beijing today. See you on Monday.

**Pull request merged**